enhancements/authentication/configuring-webhook-token-authenticators.md

```diff
@@ -97,13 +100,31 @@ secret's contents to files and specify these in the kube-apiserver configuration
 
 #### Backend - Integrated OAuth Server Token Authentication
 
+0. cluster-authentication-operator adds a new `system:openshift:openshift-token-validation`
+   cluster role to its manifests:
+   ```yaml
+   rules:
+   - nonResourceURLs:
+     - "/tokenvalidation"
```

Comment:
What makes this call? The naive implementation to avoid your picture will defeat the purpose of the cache.

How tight is the cache to the expiry?  It seems to make more sense to have cache be no larger than 10% of the total expiry and call it a day.

Reply:
This came from the original token validation implementation at oauth-apiserver, that would be just a `create tokenreviews.oauth.openshift.io` today.

I am open to having the KAS token cache being a portion of the token timeout (!= expiry), the point of this enhancement amendment is to point out that the default behavior might make users be quite surprised with their tokens unexpectedly going invalid.